### ISSUE TITLE
Vivid Voidpets with the variant 0 (Tartareans) look like default Voidpets

### ISSUE THREAD

**Issue by blue-there** [voidpet/bugs#404]
**Describe the bug**
Vivid Voidpets with the variant 0 (Tartareans) look the same as non-vivid Voidpets.

**To Reproduce**
1. Go to any User with the Tartarean
-> It has the default white eyes

OR (because its hard to find someone with a Tartarean)
1. Intercept the response of a getPet request with a tool like "Burp Suite" and modify the variant-id in the response
-> It has the default white eyes

**Expected behavior**
The red eyes like seen in the vivid variants image or defined in the vivid-color-map

**Screenshots**
![grafik](https://user-images.githubusercontent.com/99291290/162750017-43b5683c-7d52-49d3-8346-2272d8c0a3f2.png)

**Why?**
https://voidpet.com/_next/static/chunks/594-ad11f1e7bf75ab441402.js
In the image below the function r has the arguments C, L and a.
If C (the vivid-variant) is 0 the function just returns L (the default value) and not the values defined in the vividColorMap.
a is irrelevant here because its undefined most of the time.
![grafik](https://user-images.githubusercontent.com/99291290/162757955-00768ad4-771f-4cd6-b7b5-48f23fa6670a.png)

**Platforms:**
 - OS: IOS, Windows
 - Browser: Safari, Chrome, Firefox

**Additional context**
If Ben is a troll this might be **intended** to make Tartareans even more rare

**Comment by SoulessAdam** [comment]
+1, editing a server response through burp to change a vp's vivid variant to 0, renders a normal vp, if you catch a vp with variant 0 it shows a tart badge but no updated look.

Example server response:
```
"data":{"updateMyLocation":{"location":{"id":21,"layer":1,"zone":4,"name":"Ghost Road south","left":null,"right":null,"up":22,"down":19,"things":[{"id":"edb6c1a3-9bd4-432b-9e23-bb2c7938bca8","name":"Wild Lonely","stage":1,"speciesId":6,"level":1,"source":3,"isVivid":false,"variant":15,"frameSize":"54%","items":[],"theTotal":81,"__typename":"WildPet"}],"__typename":"WildLocationField"},"__typename":"LocationResponse"}}
``` 
by editing the isVivid variable and variant, we can trick clients into believing that it has a tart or any other vivid. This renders all vivids as their vivid form BUT tarts, meaning the tart variant is not displayed correctly. 

tl;dr: from what i can see, i think you might be right about us getting trolled.

**Comment by benawad** [comment]
good catch, fixed it, thanks
